### DISCUSSION TITLE
Create your own Validationattribute in ASP.NET Core

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#56] [General]
# Create your own Validationattribute in ASP.NET Core

In this small blog post, I will show you how to create your own Validation attribute in ASP.NET Core to tailor-made your validation rules.

https://steven-giesel.com/blogPost/004506a9-4c70-483d-a14f-3a56086fe6f1

**Comment by jarmyo** [comment]
Thanks!
